lib/resources/r4/appointment_patch.yaml

```diff
@@ -63,6 +63,32 @@ operations:
         <li>CodeableConcept.text is required and CodeableConcept.coding, if present, will be ignored</li>
         <li>When the value array is empty, reasonCode will be removed</li>
       </ul>
+  - name: add-cancelationReason
```

Comment:
we need to add this operation [here](https://github.com/cerner/fhir.cerner.com/blob/main/lib/resources/example_json/r4_examples_appointment.rb#L370)  as well:

Reply:
Updated the new operation here: https://github.com/cerner/fhir.cerner.com/pull/593/commits/4b69d086d4851d13bf49acff5e5f4b7c8106dba2

Reply:
(Also updated screenshots of changes before merged)